Libplanet/Libplanet.csproj

```diff
@@ -36,7 +36,7 @@ https://docs.libplanet.io/</Description>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\$(AssemblyName).xml</DocumentationFile>
-    <TargetFrameworks>netstandard2.0;netstandard2.1;netcoreapp3.1</TargetFrameworks>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
IIRC these multiple target frameworks purpose to derive benefits for .NET Core programs, especially for lib9c & NineChronicles.Standalone.

Reply:
Even if only netstandard2.0 is provided, it seems that it can be used in other projects.

Reply:
Yeah, .NET Standard 2.0 is compatible with both Mono and .NET Core, because that's why .NET Standard exists, but at the same time [.NET Standard 2.0 assemblies are optimized for neither Mono nor .NET Core][1]. For example, [*Microsoft.Bcl.AsyncInterfaces*][2], which is one of our indirect dependencies, is basically not necessary for .NET Standard 2.1 or .NET Core 3 as they have built-in `IAsyncEnumerable<T>` & others, so using `IAsyncEnumerable<T>` & others in the NuGet version of *Microsoft.Bcl.AsyncInterfaces* has performance impact in these newer runtimes AFAIK.

[1]: https://andrewlock.net/converting-a-netstandard-2-library-to-netcore-3/#upgrading-a-net-standard-2-0-library-to-net-core-3-is-it-necessary-
[2]: https://www.nuget.org/packages/System.Linq.Async